src/components/AcceleratorDeliverableView/QuestionsDeliverableCard.tsx

```diff
@@ -98,60 +102,63 @@ const QuestionBox = ({
     setEditingId(variable.id);
   }, [setEditingId, variable]);
 
+  const waitAndReload = useCallback(() => {
+    setTimeout(() => reload(), 500);
```

Comment:
For some reason, the BE can return not the latest workflow right after an update. This is a quick workaround for now.